packages/react/menu/src/Menu.tsx

```diff
@@ -516,15 +565,21 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
                   {...contentProps}
                   ref={composedRefs}
                   style={{ outline: 'none', ...contentProps.style }}
-                  onKeyDownCapture={composeEventHandlers(
-                    contentProps.onKeyDownCapture,
-                    typeaheadProps.onKeyDownCapture
-                  )}
-                  // focus first/last item based on key pressed
                   onKeyDown={composeEventHandlers(contentProps.onKeyDown, (event) => {
-                    const content = contentRef.current;
+                    // perform typeahead search
+                    const modifierKey = event.ctrlKey || event.altKey || event.metaKey;
+                    if (event.key.length === 1 && !modifierKey) {
+                      const typeaheadKeyDownEvent = new CustomEvent(TYPEAHEAD_SEARCH, {
+                        bubbles: true,
```

Comment:
Do we need this to `bubble` if we're always calling `event.stopPropagation`? Looks like we're dispatching and listening on `content` so seems we can remove the latter if we set this to `false`?

Reply:
Yeah I thought this too, but we need to dispatch on `event.target` (menu item) and bubble it up to `content`

Reply:
Ah but why do we need to dispatch on `event.target`? I can't see `event.target` being referenced in `handleTypeaheadSearch`. Would dispatching on `event.currentTarget` suffice? 💭

Reply:
In fact, if we can dispatch on `event.currentTarget` then we wouldn't need `event.stopPropagation` and looks like we wouldn't need the custom event then either? I feel like I must be missing something 🤔

Reply:
It needs to be `event.target` so that it is scoped correctly to the menu of origin. We can’t use standard `onKeyDown` events on `content` without stopping the source events propagation (which we try not to do) else they bubble all the way up the stack. Instead we are using a `customEvent` to give us the relationship between the item and menu of origin so that the `newItem` is taken from the single, correct collection.

Reply:
It was confusing to think it through again. Because we've decided to not stop prop in the traditional way, what's actually happening is that the handlers will fire for every menu in the stack because `onKeyDown` will still bubble. So `event.target` is our way to qualify against that.

Reply:
oh my, this is hurting my brain... can we get on a call quick so you can walk me through this?

Reply:
Just trying to get my head around this one again and this is what my thinking was:

![image](https://user-images.githubusercontent.com/175330/120471720-f654d000-c39c-11eb-9b5f-a5c9fc09ee04.png)

This appears to work okay but I have no idea if I'm catching all the edge-cases here. Does this cover it?

Reply:
Tested and works great, such a simpler solution.